.github/workflows/eks-add-on-integ-test.yml

```diff
@@ -11,12 +11,12 @@ on:
       addon_name:
         required: true
         type: string
-        default: "amazon-cloudwatch"
+        default: "amazon-cloudwatch-observability"
         description: "EKS addon name"
       addon_version:
         required: true
         type: string
-        default: "v0.1.0-eksbuild.4"
```

Comment:
will we need this update this for every add-on release

Reply:
This is to just provide default , when we execute this integration test, the MCM operator need to pass the releasing the addon version as workflow input to validate

Reply:
gotcha thanks